app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009-2014 Martin Szulecki
+		   2008-2010 Jonathan Beck
+		   2009 Nikias Bassen
+		   2008 Matt Colyer
+		   "
+LICENSE="GNU GPL v2
+		 GNU LGPL v2.1"
```

Comment:
There should be only one tab on the beginning of each line (for COPYRIGHT and LICENCE) except the very first line. So we should have this:
```
LICENSE="GNU GPL v2
	GNU LGPL v2.1"
```
Funny "rules", isn't it? Well, you'll get used to it after a couple of recipes ;-)